host/src/server/utils.rs

```diff
@@ -0,0 +1,95 @@
+use crate::server::api::{v2, v3};
+use raiko_reqpool::Status;
+use raiko_tasks::TaskStatus;
+
+pub fn to_v2_status(result: Result<Status, String>) -> v2::Status {
```

Comment:
nice organization, just a reminder, please make sure the response is consistent.

Reply:
Yes. Our test already ensure this.